src/buzzer-element.ts

```diff
@@ -0,0 +1,95 @@
+import { css, customElement, html, LitElement, property } from 'lit-element';
+
+/**
+ * Renders a piezo electric buzzer.
+ */
+@customElement('wokwi-buzzer')
+export class BuzzerElement extends LitElement {
+  /**
+   * Boolean representing if an electric signal is coming in the buzzer
+   * If true a music note will be displayed on top of the buzzer
+   */
+  @property() value = false;
```

Comment:
This is really cool 😎 maybe change the prop name to "hasSignal" so it would be more informative?

Reply:
Done :+1: